client/assets/stylesheets/_wp-base-styles-overrides.scss

```diff
@@ -0,0 +1,80 @@
+.blaze-pro,
+.is-blaze-pro {
+	.form-text-input-core-styles:not(.is-error) {
+		--wp-admin-theme-color: var(--color-gray);
+	}
+}
+
+.woo.is-woo-passwordless {
+	input[type='text'],
+	input[type='password'] {
+		&.form-text-input-core-styles {
+			/* stylelint-disable-next-line scales/radii */
+			border-radius: 8px;
+			font-size: 1rem;
+			height: 48px;
+			padding: 8px 16px;
+
+			&:not(:focus, :disabled) {
+				border-color: var(--studio-gray-30);
+			}
+
+			&:focus {
+				border-width: 1.5px;
+			}
+
+			&:not(.is-error) {
+				--wp-admin-theme-color: var(--woo-purple-50);
+			}
+		}
+	}
+}
+
+.is-akismet {
+	.form-text-input-core-styles:not(.is-error) {
+		--wp-admin-theme-color: var(--color-akismet);
+	}
+}
+
+.a8c-for-agencies {
+	.form-text-input-core-styles:not(.is-error) {
+		--wp-admin-theme-color: var(--color-primary-40);
+	}
+}
+
+.jetpack-cloud {
+	.form-text-input-core-styles:not(.is-error) {
+		--wp-admin-theme-color: var(--studio-jetpack-green-40);
+	}
+}
+
+.layout.is-jetpack-login:not(.is-woocommerce-core-profiler-flow) {
+	.form-text-input-core-styles:not(.is-error) {
+		--wp-admin-theme-color: var(--studio-jetpack-green-50);
+	}
+}
+
+.layout.is-section-login.is-grav-powered-client {
+	input[type='text'],
+	input[type='password'] {
+		&.form-text-input-core-styles {
+			&:not(:focus, :disabled) {
+				border-color: #9ea4a8;
```

Comment:
If there's no variable for this color, maybe we can add an inline comment explaining if there's at least a reference to is in the design specs?

Reply:
I tried to maintain the original, but I think there's a strong case here for using the default.

Reply:
Removed in [4cae5aa](https://github.com/Automattic/wp-calypso/pull/103616/commits/4cae5aa78a2b33c7e1b041b9e222735c5eae3df4)